File: lib/models/DanhMucYeuThich.dart
class DanhMucYeuThich {
  DanhMucYeuThich({
    required this.danhMucYeuThichId,
    required this.khachHangId,
    required this.sanPhamId,
    required this.thoiGian,
  });

  final int? danhMucYeuThichId;
  final String? khachHangId;
  final int? sanPhamId;
  final DateTime? thoiGian;

  factory DanhMucYeuThich.fromJson(Map<String, dynamic> json) {
    return DanhMucYeuThich(
      danhMucYeuThichId: json["danhMucYeuThichId"],
      khachHangId: json["khachHangId"],
      sanPhamId: json["sanPhamId"],
      thoiGian: DateTime.tryParse(json["thoiGian"] ?? ""),
    );
  }
}

File: API_DBFruits/API_DBFruits/Models/DanhMucYeuThich.cs
using System;
using System.Collections.Generic;

namespace API_DBFruits.Models;

public partial class DanhMucYeuThich
{
    public int DanhMucYeuThichId { get; set; }

    public string? KhachHangId { get; set; }

    public int? SanPhamId { get; set; }

    public DateOnly ThoiGian { get; set; }

    public virtual User? KhachHang { get; set; }

    public virtual SanPham? SanPham { get; set; }
}
